tools/cabana/videowidget.cc

```diff
@@ -128,7 +142,16 @@ void VideoWidget::updatePlayBtnState() {
 // Slider
 Slider::Slider(QWidget *parent) : timer(this), thumbnail_label(this), QSlider(Qt::Horizontal, parent) {
   timer.callOnTimeout([this]() {
-    timeline = can->getTimeline();
+    auto unsortedTimeline = can->getTimeline();
+
+    std::vector<std::tuple<int, int, TimelineType>> sortedTimeline;
+
+    std::copy(unsortedTimeline.begin(), unsortedTimeline.end(), std::back_inserter(sortedTimeline));
+
+    std::sort(sortedTimeline.begin(), sortedTimeline.end(), sortTimelineBasedOnEventPriority);
+
+    timeline = sortedTimeline;
```

Comment:
can simplify to:
timeline = can->getTimeline();
std::sort(timeline.begin(), timeline.end(), sortTimelineBasedOnEventPriority);